README.md

```diff
@@ -5,7 +5,9 @@ This project requires Node v16.x.x (npm v8.x.x). You can use Node Version manage
 curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
 nvm install 16
 ```
-...and then you can do `npm install` etc as below.
+...and then you should create an `.env` file and copy everything from `.env.example` into `.env`. \
+
+...Finally you can do `npm install` etc as below.
```

Comment:
Yes, good idea, but move the new instructions below the npm install line (since it is currently between the nvm and npm lines and isn't related to those).

Reply:
I also like this addition.